### ISSUE TITLE
Getting "unexpected end of file" when attempting to untarr the abc_0.0.1-alpha8_linux_amd64.tar.gz

### ISSUE THREAD

**Issue by hernandezb-source** [abcxyz/abc#197]
### TL;DR

Getting "unexpected end of file" when attempting to untarr the abc_0.0.1-alpha8_linux_amd64.tar.gz

### Expected behavior

untarring file

### Observed behavior

curl https://github.com/abcxyz/abc/releases/download/v0.0.1-alpha8/abc_0.0.1-alpha8_linux_amd64.tar.gz | tar -xzv abc
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now

### Environment Details

```markdown
cloudtop
```

**Comment by drevell** [comment]
Thanks for sending this.

I think this was a copy-paste mistake. `curl` should be run as `curl -sSL`. The example in the docs show that, but I'm sending a PR to make this more obvious.

**Comment by hernandezb-source** [comment]
Thank you,

**Comment by drevell** [comment]
> Thank you, I would also suggest updating the text "Pick the most recent release that isn't an -alpha or -rc or anything, just vX.Y.Z" to reflect that only -alpha releases exist.

Good point. We're intending to release a 0.0.1 very soon (tomorrow?), so hopefully we won't need this
